src/network/serverpackethandler.cpp

```diff
@@ -761,12 +763,13 @@ void Server::handleCommand_ChatMessage(NetworkPacket* pkt)
 		message += (wchar_t)tmp_wchar;
 	}
 
-	RemotePlayer *player = m_env->getPlayer(pkt->getPeerId());
+	session_t peer_id = pkt->getPeerId();
+	RemotePlayer *player = m_env->getPlayer(peer_id);
 	if (player == NULL) {
-		errorstream << "Server::ProcessData(): Canceling: "
-				"No player for peer_id=" << pkt->getPeerId()
-				<< " disconnecting peer!" << std::endl;
-		DisconnectPeer(pkt->getPeerId());
+		errorstream <<
+			"Server::ProcessData(): Canceling: No player for peer_id=" <<
```

Comment:
i don't think this way to call errorstream is compliant with our guidelines no ?

Reply:
The preferred style is
```c++
    errorstream
        << "foobar" << ...
```
but I don't think this style is disallowed by the guidelines.

Reply:
I don't find a [guideline](https://dev.minetest.net/Code_style_guidelines#Spaces) which disallows this.
I've moved the message to a separate line so that the constant string is not separated, which should make it easier to read.

Reply:
Those messages are lots of duplicate code, but fixing that is not part of this issue.

Reply:
> The preferred style is […]

In Lua I always do the string concatenation the other way round, so I've did it here, too.